cmake/CatkinBuild.cmake

```diff
@@ -37,22 +33,29 @@ add_library(nabo
   ${NABO_SRC}
   ${NABO_HEADERS}
 )
-
-target_include_directories(nabo
-  PUBLIC
-    ${CMAKE_SOURCE_DIR}
-    ${CMAKE_SOURCE_DIR}/nabo
-  SYSTEM
-    ${EIGEN3_INCLUDE_DIR}
-    ${Boost_INCLUDE_DIRS}
-    ${OpenMP_CXX_INCLUDE_DIRS}
+target_include_directories(nabo PUBLIC
+  ${CMAKE_SOURCE_DIR}
+  ${CMAKE_SOURCE_DIR}/nabo
 )
-
-target_link_libraries(nabo
+target_include_directories(nabo SYSTEM PRIVATE
+  ${EIGEN3_INCLUDE_DIR}
+  ${Boost_INCLUDE_DIRS}
```

Comment:
Boost isn't used in the core library, just for the python bindings. So remove it from here.